src/Modules/ModulesGraded.jl

```diff
@@ -1562,28 +1566,40 @@ twist:  -6  -5  -4  -3  -2  -1   0   1   2
 ------------------------------------------
 chi:     *   *   *   4   -   -   1   -   *
 
-julia> tbl[0, -6]
-70
+julia> tbl[3, 2]
+6
 
 julia> tbl[2, 0]
 1
 
+julia> sheaf_cohomology(M, -9, 5)
+twist:   -9   -8   -7   -6   -5   -4   -3   -2   -1    0    1    2    3    4    5
+---------------------------------------------------------------------------------
+0:      280  189  120   70   36   15    4    -    -    -    -    -    *    *    *
+1:        *    -    -    -    -    -    -    -    -    -    -    -    -    *    *
+2:        *    *    -    -    -    -    -    -    -    1    -    -    -    -    *
+3:        *    *    *    -    -    -    -    -    -    -    -    6   20   45   84
+---------------------------------------------------------------------------------
+chi:      *    *    *   70   36   15    4    -    -    1    -    6    *    *    *
+```
+
+```jldoctest
 julia> R, x = polynomial_ring(QQ, "x" => 1:5);
 
-julia> R, x = grade(R);
+julia> S, _  = grade(R);
```

Comment:
You might want to use graded_polynomial_ring here to adhere to the standard notation?
(also at other places)